docs/dcomibmtoolsattachretry.md

```diff
@@ -0,0 +1,46 @@
+<!--
+* Copyright (c) 2017, 2024 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# -Dcom.ibm.tools.attach.retry
+
+**Windows&trade; only**
+
+This option specifies the number of times the `jcmd` tool retries attaching to a running VM when the tool encounters the `SocketException` error.
+
+## Syntax
+
+        -Dcom.ibm.tools.attach.retry=<number_of_retries>
+
+The default value for `<number_of_retries>` is 3.
+
+## Explanation
+
+When the `jcmd` tool sends a command to a running VM, the command might throw the `Socket Exception` error in case of issues, such as a deadlock or memory management issues. Instead of failing the attaching request, you can specify the number of times the tool retries attaching to the target VM with the `-Dcom.ibm.tools.attach.retry` system property.
```

Comment:
> the command might throw the `Socket Exception` error in case of issues,  such as a deadlock or memory management issues

A deadlock or memory management issues are not the typical causes.
As per https://learn.microsoft.com/en-us/previous-versions/ms832256(v=msdn.10)?redirectedfrom=MSDN, following SocketException is seen occasionally at Windows `java.net.SocketException: Software caused connection abort: socket write error`.

Reply:
Can we say " ... in case of issues, such as a network failure or a connection reset"?

Reply:
This sounds right.